static/js/templates.js

```diff
@@ -1,3 +1,14 @@
+let boilerplates, snippets, featured_boilerplates
+const grabTemplates = async () => {
```

Comment:
does this need to be async? i see this has been marked async and await is used on L#25, but there doesn't appear to be any async code in here